### ISSUE TITLE
CLI: Support cookies.txt

### ISSUE THREAD

**Issue by jtagcat** [gildas-lormeau/SingleFile#574]
**Is your feature request related to a problem? Please describe.**
It's annoying and time-consuming having to convert a list of cookies to CLI arguments.

**Describe the solution you'd like**
Support injecting cookies on the CLI with either cookies.txt or cookies.json.

**Comment by jtagcat** [comment]
Made a dumb little script meanwhile.
```
from numpy import loadtxt

lines_numpy = loadtxt("cookies.txt", comments="#", delimiter="\t", unpack=False, dtype=str)
lines = lines_numpy.tolist()

arguments_list = []

for line in range(len(lines)):
    cookie = lines[line]
    # comments for next line                name              value             domain            path              expires           httponly?         secure?  (samesite) (url)
    arguments_list.append("--browser-cookie '" + cookie[5] + "," + cookie[6] + "," + cookie[0] + "," + cookie[2] + "," + cookie[4] + "," + cookie[3] + "," + cookie[1] + "'")

arguments_all = ' '.join(arguments_list)
print(arguments_all)
```

**Comment by gildas-lormeau** [comment]
Thank you for the suggestion. I didn't think of this feature because it has been years since browsers stopped storing cookies in a text file. What tools do you use to create a cookies.txt/cookies.json file?

**Comment by jtagcat** [comment]
I don't remember anything about where I got the json export from.

Pretty much all tools I have used use cookies.txt (what apparently in selenium can be json, in .txt!). I use https://github.com/lennonhill/cookies-txt normally.

***
The script above is kind of outdated, since I modified it to give me a docker command on the fly (fresh set of cookies keeps the captcha monster away). (Was too lazy to implement send-captcha-to-somewhere-and-get-a-token-back things.)
